### PR TITLE
Giveitem with player dropdown menu

### PR DESCRIPTION
**Pull request by i-kulgu**
## Description

With this PR you can give an item more controlled.
You'll get a list of nearby players and when you choose a player it will give the item to that selected player.

Preview:
https://www.youtube.com/watch?v=WtpmP7GfGF0

## Checklist

- [X ] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [X ] My code fits the style guidelines.
- [X ] My PR fits the contribution guidelines.

### CODE REVIEW

**Review by Irishstevie — APPROVED** [review]
Love Just tested Without an issue <3

**Comment by M4RFY** [comment]
i like the idea

**Review by M4RFY — APPROVED** [review]
Perfectly Working

**Review by Twikii — APPROVED** [review]
Works well

File: html/js/app.js [hunk 1]
```diff
@@ -2915,14 +2953,20 @@ $("#item-give").droppable({
         setTimeout(function() {
             IsDragging = false;
         }, 300);
+        $("#item-amount").fadeOut(0);
```
Comment (zwkf) [inline html/js/app.js:2956]: `fadeOut()` with `0` duration seems a little odd. 
Maybe `hide()` would be better suited here?

http://api.jquery.com/hide/

File: html/js/app.js [hunk 2]
```diff
@@ -2931,3 +2975,13 @@ $("#item-give").droppable({
         );
     },
 });
+
+$(document).on("click", "#inv-dialog-return", function(e) {
+    e.preventDefault();
+    $("#item-amount").fadeIn(0);
```
Comment (zwkf) [inline html/js/app.js:2981]: Refer to above comment about hide/show

File: html/js/app.js [hunk 3]
```diff
@@ -2886,6 +2921,9 @@ var requiredItemOpen = false;
                 case "toggleHotbar":
                     Inventory.ToggleHotbar(event.data);
                     break;
+                case "NearPlayers":
+                    Inventory.NearPlayers(event.data)
+                    break
```
Comment (zwkf) [inline html/js/app.js:2926]: missing semi colon

**Review by zwkf — CHANGES_REQUESTED** [review]
Left some comments

**Comment by i-kulgu** [comment]
> Left some comments

Fixed all of the comments and added a configurable option to hide player names upon giving item.

**Comment by zwkf** [comment]
Note to self: test this

**Pull request closed without merging**